modules/atomic-widgets/atomic-controls/atomic-section.php

```diff
@@ -0,0 +1,53 @@
+<?php
+namespace Elementor\Modules\AtomicWidgets\AtomicControls;
+
+use JsonSerializable;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly
+}
+
+class Atomic_Section implements JsonSerializable {
+	private array $value = [];
+
+	public static function make(): self {
+		return new static();
+	}
+
+	public function set_label( string $label ): self {
+		$this->value['label'] = $label;
+
+		return $this;
+	}
+
+	public function set_description( string $description ): self {
+		$this->value['description'] = $description;
+
+		return $this;
+	}
+
+	public function set_items( array $items ): self {
+		$this->value['items'] = $items;
+
+		return $this;
+	}
+
+	public function get_label(): string {
+		return $this->value['label'];
```

Comment:
```suggestion
		return $this->value['label'] ?? null;
```

maybe? 
also others

Reply:
I think we should decide rather we want the section and control props to be accessible or not, if we don't want them to be accessible we could simply delete this method.